hyper/transport/grpc/restore.go

```diff
@@ -0,0 +1,38 @@
+package grpc
+
+import (
+	"context"
+	"os"
+
+	"github.com/isard-vdi/isard/common/pkg/grpc"
```

Comment:
This should be in the "external" imports group (with "grpc/codes" and all the others)

Reply:
I don't catch this